inference-engine/tests/functional/plugin/cpu/test_utils/cpu_test_utils.cpp

```diff
@@ -55,9 +55,13 @@ cpu_memory_format_t CPUTestsBase::cpu_str2fmt(const char *str) {
 }
 
 std::string CPUTestsBase::fmts2str(const std::vector<cpu_memory_format_t> &fmts) {
+    return fmts2strWithPrefix(fmts, "cpu:");
+}
+
+std::string CPUTestsBase::fmts2strWithPrefix(const std::vector<cpu_memory_format_t> &fmts, const std::string &prefix) {
```

Comment:
Isn't it better to use just one function `fmts2str` with two parameters where the last one has default value?

Reply:
Had the same idea, but from my opinion it does not fit here well, since the default value would be "cpu:" and not just empty string. But maybe it is still better to use second parameter with default value, I am not sure.

Reply:
Ok, I see. How about this? Make `fmts2str ` taking two parameters: format vector and prefix. Then, use it in `getTestCaseName` with an empty prefix and in `makeCPUInfo` and other relevant contexts (if any exist) with the `cpu:` prefix. Thus there is no need for the separate `fmts2strWithPrefix` function.

Reply:
Done